sqlstream/reader.go

```diff
@@ -16,7 +16,7 @@ type DataRows interface {
 	Scan(dest ...interface{}) error
 }
 
-type sqlReader struct {
+type SQLReader struct {
```

Comment:
You are right, it makes no sense since we are creating it via the NewSQLReader function.